src/main/java/com/code972/elasticsearch/plugins/DictReceiver.java

```diff
@@ -12,25 +12,24 @@
  */
 public class DictReceiver {
     private static String home = System.getProperty("user.home");
-    private static String[] gZipFilePaths = {"plugins/analysis-hebrew/dictH.dict", "/var/lib/hebmorph/dictH.dict", home + "/hebmorph/dictH.dict"};
-    private static String[] hspellFilePaths = {"plugins/analysis-hebrew/hspell-data-files/", "/var/lib/hspell-data-files/", home + "/hspell-data-files/"};
-    private static String DEFAULT_HSPELL_PREFIX = HSpellLoader.PREFIX_NOH;
+    private static String[] filePaths = {"plugins/analysis-hebrew/dictionary.dict", "/var/lib/hebmorph/dictionary.dict", home + "/hebmorph/dictionary.dict", 
+                                            "plugins/analysis-hebrew/hspell-data-files/", "/var/lib/hspell-data-files/", home + "/hspell-data-files/"};
     private static DictHebMorph dict = null;
 
     public static DictHebMorph getDictionary() {
         if (dict == null) {
-            dict = setDictionary();
+            dict = setDefaultDictionary();
         }
         return dict;
     }
 
-    public static boolean setHebmorphDictionary(String dictheb_path) {
-        if (dictheb_path != null) {
-            File file = new File(dictheb_path);
+    public static boolean setDictionary(String path) {
+        if (path != null) {
+            File file = new File(path);
             if (file.exists()) {
                 try {
-                    System.out.println("Successfully loaded from: " + dictheb_path);
-                    dict = DictionaryLoader.loadDicAndPrefixesFromGzip(dictheb_path);
+                    dict = DictionaryLoader.loadDictFromPath(path);
+                    System.out.println("Successfully loaded from: " + path);
```

Comment:
There isn't any actual support in elasticsearch for plugin-logging. Do you want me to implement a log specifically for that?